pkgs/development/libraries/qtutilities/default.nix

```diff
@@ -27,7 +26,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     homepage = "https://github.com/Martchus/qtutilities";
     description = "Common C++ classes and routines used by @Martchus' applications featuring argument parser, IO and conversion utilities";
```

Comment:
I'd like to note that this is actually the description of c++utilities and not the one of qtutilities.

Reply:
Thanks @Martchus, will be fixed on the next update!